### DISCUSSION TITLE
Suggested Road Map 📍

### DISCUSSION THREAD

**Discussion by fredygerman** [fredygerman/umeme-upo#8] [Announcements]
## Our Suggested Roadmap 🗺️ (this is open for discussion, please share your point of views)

We have a clear roadmap to enhance your power information experience:

1. **Enhanced Viewing Options**:
   - *Map and Timeline Views*: These can give a visual snapshot of power status in different locations. This means users can quickly assess and plan around power availability.

2. **TANESCO Data Integration**:
   - *Leveraging TANESCO Timetable*: By using into the TANESCO timetable data, we can provide users power status information according to the TANESCO timetable. This will be great but we will need to mark the results based on this due to the timetable not being very accurate.

3. **Community Reporting**:
   - *Your Reporting Power*: Allows users and our community members to report power outages and restorations. This will help maintain a comprehensive and accurate power status database.

4. **IoT Devices**:
   - *Real-Time Data Precision*: To take our accuracy to the next level, we can explore the use of IoT devices in various areas. These devices will enable real-time data updates and ensure the most precise power status information for you. 📡

Together, we're building a more informed and connected Dar es salaam. Stay tuned for updates on our progress! ⚙️🔌

For now Head to the issues / discussions for each of the Items listed above and share your comments and ideas.

**Comment by neicore** [comment]
congrats @fredygerman for the initiative. Have you been in touch with Tanesco or someone at Tanesco to see if they have like a monitoring system they be willing to share some of their APIs to the public?

**Reply by fredygerman** [reply]
I haven't gotten in touch with anyone from Tanesco but will do that since it will make things much more straight forward. Thanks for the idea.
